packages/firebase_core/ios/firebase_core.podspec

```diff
@@ -18,4 +18,11 @@ A new flutter plugin project.
   s.dependency 'Firebase/Core'
   s.ios.deployment_target = '8.0'
   s.static_framework = true
+
+  s.prepare_command = <<-CMD
+    PUBSPEC_VERSION=`cat ../pubspec.yaml | grep version: | sed 's/version: //g'`
```

Comment:
It's possible to read the yaml file via the default Ruby yaml library that comes with OSX

```ruby
require 'yaml'
pubspec = YAML.load_file(File.join('..', 'pubspec.yaml'))

# ...

Pod::UI.puts "This plugin version is '#{pubspec['version']}'"
```